olp-cpp-sdk-authentication/test/unit/AuthenticationProductionTests.cpp

```diff
@@ -53,8 +53,8 @@ class AuthenticationOnlineProductionTest : public ::testing::Test {
     client = std::make_unique<AuthenticationClient>();
     client->SetTaskScheduler(
         olp::client::OlpClientSettingsFactory::CreateDefaultTaskScheduler());
-    client->SetNetwork(
-        olp::client::OlpClientSettingsFactory::CreateDefaultNetworkRequestHandler());
+    client->SetNetwork(olp::client::OlpClientSettingsFactory::
```

Comment:
We will face the same issues here as with dataservice-read/write online tests. I would prefer you introduce a static network and instantiate it in `static void SetUpTestSuite()`